backend/fms_core/templates.py

```diff
@@ -174,11 +174,11 @@
 }
 
 NORMALIZATION_TEMPLATE = {
-  "identity": {"description": "Template to perform normalization", "file": static("submission_templates/Normalization_v3_10_0.xlsx")},
+  "identity": {"description": "Template to perform normalization", "file": static("submission_templates/Normalization_v3_14_0.xlsx")},
   "sheets info": [
       {
         'name': 'Normalization',
-        'headers': ['Sample Name', 'Source Container Barcode', 'Source Container Coord', 'Robot Source Container', 'Robot Source Coord',
+        'headers': ['Sample Name', 'Source Container Barcode', 'Source Container Coord', 'Sample Initial Concentration', 'Robot Source Container', 'Robot Source Coord',
```

Comment:
Placement of the field is not great in between the source container and its robot equivalent. Rename (Initial Conc. (ng/uL)) and move the field after Source Depleted. Do not forget to update "Conc. (ng/uL)" formula.